engine/jobmaster/dm/metadata/job.go

```diff
@@ -110,6 +112,11 @@ func (jobStore *JobStore) UpdateStages(ctx context.Context, taskIDs []string, st
 	}
 
 	job := state.(*Job)
+	if len(taskIDs) == 0 {
```

Comment:
in which path this is empty 🤔

Reply:
If not specify tasks. e.g. `operate-job job-id pause`